Makefile

```diff
@@ -17,6 +17,12 @@ clean:
 
 composer:
 	docker-compose run --rm composer bash -c "composer install --no-scripts"
+	touch vendor/simplesamlphp/simplesamlphp/config/authsources.php
```

Comment:
I may have asked this on another PR, but why is the `touch` neeeded?

Reply:
It's so that the volume mounts that docker-compose will do (after this) have a file to mount into the container (iirc). Otherwise, it creates empty directories there and hits an error. I fought through this a month ago and ended up at this solution being workable, where other options I tried didn't.